modules/packages/TOML.chpl

```diff
@@ -810,7 +843,8 @@ used to recursively hold tables and respective values
           when fieldEmpty {
             throw new owned TomlError("Keys must have a value");
           }
-          when fieldDate {
+          // TODO: writef for fieldDate
```

Comment:
Were you planning on implementing this TODO in this PR?

Reply:
Yes, I've made the changes and is ready to merge. Thank you for pointing that out.